docs/settings.rst

```diff
@@ -168,6 +168,15 @@ Setting name (followed by default value, if any)
                                                                                  is also in ``ARTICLE_PATHS`` or ``PAGE_PATHS``. If you are trying
                                                                                  to publish your site's source files, consider using the
                                                                                  ``OUTPUT_SOURCES`` setting instead.
+``STATIC_CREATE_LINKS = False``                                                  Create links instead of copying files. If the content and output
```

Comment:
Hm, I think you're thinking of statically linked libraries, no? How about `STATIC_CREATE_HARDLINKS`?